src/corehost/cli/dll/CMakeLists.txt

```diff
@@ -67,3 +67,4 @@ if (WIN32 AND CLI_CMAKE_PLATFORM_ARCH_ARM)
     target_link_libraries(hostpolicy shell32.lib)
 endif()
 
+install_library_and_symbols (hostpolicy)
```

Comment:
This was copied from the other cmakelists.txt files.  I can remove the newlines if it's annoying.